src/pyrokinetics/gk_code/gkw.py

```diff
@@ -0,0 +1,418 @@
+import copy
+import warnings
+from typing import Any, Dict, Optional
+
+import f90nml
+import numpy as np
+import pint
+from cleverdict import CleverDict
+
+from ..constants import deuterium_mass, electron_mass, pi, sqrt2
+from ..local_geometry import LocalGeometry, LocalGeometryMiller, default_miller_inputs
+from ..local_species import LocalSpecies
+from ..normalisation import SimulationNormalisation as Normalisation
+from ..normalisation import convert_dict, ureg
+from ..numerics import Numerics
+from ..templates import gk_templates
+from ..typing import PathLike
+from .GKInput import GKInput
```

Comment:
```suggestion
from .gk_input import GKInput
```

This should help with the test